packages/swingset-runner/bin/runner-debug

```diff
@@ -4,7 +4,11 @@
  * Simple boilerplate program providing linkage to launch an application written using modules within the
  * as yet not-entirely-ESM-supporting version of NodeJS.
  */
-import lmdb from 'node-lmdb';
+
+// LMDB bindings need to be imported before lockdown.
```

Comment:
why do LMDB bindings need to be imported before lockdown?

Reply:
Weird native code stuff.